CHANGELOG.md

```diff
@@ -52,6 +52,14 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-SCOPE
     * #### Added
+        * API parity with NI-SCOPE 20.0
+            * `END_OF_RECORD_EVENT_TERMINAL_NAME`
+            * `ADVANCE_TRIGGER_TERMINAL_NAME`
+            * `REF_TRIGGER_TERMINAL_NAME`
+            * `START_TRIGGER_TERMINAL_NAME`
+            * `READY_FOR_ADVANCE_EVENT_TERMINAL_NAME`
+            * `READY_FOR_REF_EVENT_TERMINAL_NAME`
+            * `READY_FOR_START_EVENT_TERMINAL_NAME`
```

Comment:
These should be the Python propery names. As added, they are metadata names, which clients would not recognize unless for some obscure reason they looked at the repo.

For less ambiguity, consider typing `Session.end_of_record_event_terminal_name`. If it was a method, you'd add `()` at the end.